docs/modules/mssqlfg.md

```diff
@@ -0,0 +1,300 @@
+| `azure-sql-12-0-dr-database-pair-from-existing-primary` | Taking over an existing database upon the primary server of a previous DBMS pair, and extend it to a failover group deployment. |
+
+All the services in this module require `ENABLE_DISASTER_RECOVERY_SERVICES` to be `true` in OSBA environment variables. Besides, `azure-sql-12-0-dr-database-pair-from-existing` and `azure-sql-12-0-dr-database-pair-from-existing-primary` require `ENABLE_MIGRATION_SERVICES` to be `true`. The `azure-sql-12-0-dr-dbms-pair-registered` service allows you to provide an existing primary server and an existing secondary server for the DR deployment. For more information on each service, refer to the descriptions below.
+
+## Services & Plans
+
+### Service: azure-sql-12-0-dr-dbms-pair-registered
+
+| Plan Name | Description |
+|-----------|-------------|
+| `dbms` | Azure SQL Server DBMS-Only |
+
+#### Behaviors
+
+##### Provision
+
+Register a pair of SQL servers as a service instance: check the existence of these servers; check if the input administrator logins work. Databases with failover group can be created through subsequent provision requests using the `azure-sql-12-0-dr-database-pair` service.
+
+###### Provisioning Parameters
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| `primaryResourceGroup` | `string` | The (new or existing) resource group with which to associate new resources. | Y | |
+| `primaryLocation` | `string` | The Azure region in which to provision applicable resources. | Y | |
+| `primaryServer` | `string` | The name of your existing primary server. | Y | |
+| `primaryAdministratorLogin` | `string` | The administrator login of the primary server. | Y | |
+| `primaryAdministratorLoginPassword` | `string` | The administrator login password of the primary server. | Y | |
+| `secondaryResourceGroup` | `string` | The (new or existing) resource group with which to associate new resources. | Y | |
+| `secondaryLocation` | `string` | The Azure region in which to provision applicable resources. | Y | |
+| `secondaryServer` | `string` | The name of your existing secondary server. | Y | |
+| `secondaryAdministratorLogin` | `string` | The administrator login of the secondary server. | Y | |
+| `secondaryAdministratorLoginPassword` | `string` | The administrator login password of the secondary server. | Y | |
+| `tags` | `map[string]string` | Tags to be applied to new resources, specified as key/value pairs. | N | Tags (even if none are specified) are automatically supplemented with `heritage: open-service-broker-azure`. |
+| `alias` | `string` | Specifies an alias that can be used by later provision actions to create database pairs on this DBMS pair. | Y | |
+
+##### Bind
+
+This service is not bindable.
+
+##### Update
+
+Updates broker-stored administrator login/password in case you reset them.
```

Comment:
If the user has changed the login/password and hasn't updated the instance, will the binded application fail to work? If true, when will the user possibly change the login/password outside the broker?

Reply:
The server is actually managed outside the broker, that's why the user uses "registered" service. You know SQL server has too many advanced functionalities: auditing, threat detection, heritage, service endpoints, etc.. So, big customers prefer to managing the server themselves. It is possible for them to reset the password. As this is a "registered" service, it makes sense to update what to register, right? Also, MASB user requested this feature.

Reply:
Got it. Thanks for the explanation!